### ISSUE TITLE
Should the CWL reference runner support FTP locations for input files?

### ISSUE THREAD

**Issue by mr-c** [common-workflow-language/cwltool#933]
Such functionality could be brought over from https://github.com/common-workflow-language/cwl-tes/pull/25/files#diff-c1c308a13ce3637600a5aed5d4bfaa56R36

However, not every other CWL implementation supports FTP locations, so it could be a bad user experience.

**Comment by tetron** [comment]
If we switched to pyfilesystem (#922) and refactored all the file I/O to go through it, then FTP support would be included for free?

**Comment by mr-c** [comment]
@mr-c Yes. Though that is likely to be a lot of work, so we could shim in FtpStdAccess in the mean time.
